### DISCUSSION TITLE
Is there a way to directly redirect a string to a command (aka <<< posix shell eqivalent)?

### DISCUSSION THREAD

**Discussion by quantenzitrone** [fish-shell/fish-shell#9848] [Q&A] [answered]
In posix shell you can either do:
```sh
bc <<<"1+1"
```
or 
```sh
echo "1+1" | bc
```
In fish only the latter works, so i wondered if there is a way to directly redirect a sting to a command without using `echo` and `|`.
If not, could it be added, because it enables users to feed input to an interactive command in a convenient way.

also i know i can use `math` to do calculations in the shell i only used bc as an example here

**Comment by faho** [comment, marked as answer]
> If not, could it be added, because it enables users to feed input to an interactive command in a convenient way.

This is barely a convenience, and it's unlikely we'll add it. See https://fishshell.com/docs/current/fish_for_bash_users.html#heredocs
